docs/psa-driver-example-and-guide.md

```diff
@@ -140,23 +140,23 @@ This guide assumes you are building Mbed TLS from source alongside your project.
 
 [p256-m](https://github.com/mpg/p256-m) is a minimalistic implementation of ECDH and ECDSA on the NIST P-256 curve, specifically optimized for use in constrained 32-bit environments. It started out as an independent project and has been integrated in Mbed TLS as a PSA transparent driver. The source code of p256-m and the driver entry points is located in the Mbed TLS source tree under `3rdparty/p256-m`. In this section, we will look at how this integration was done.
 
-The Mbed TLS build system includes the instructions needed to build p256-m. To build with and use p256-m, set the macro `MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED` using `config.py`, then build as usual using make/cmake. From the root of the `mbedtls/` directory, run:
+The Mbed TLS build system includes the instructions needed to build p256-m. To build with and use p256-m, set the macro `MBEDTLS_PSA_P256M_DRIVER_ENABLED` using `config.py`, then build as usual using make/cmake. From the root of the `mbedtls/` directory, run:
 
     python3 scripts/config.py set MBEDTLS_PSA_CRYPTO_CONFIG
-    python3 scripts/config.py set MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED
+    python3 scripts/config.py set MBEDTLS_PSA_P256M_DRIVER_ENABLED
     make
 
-(You need extra steps if you want to disable the built-in implementation of ECC algorithms, which includes more features than p256-m. Refer to the documentation of `MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED` for more information.)
+(You need extra steps if you want to disable the built-in implementation of ECC algorithms, which includes more features than p256-m. Refer to the documentation of `MBEDTLS_PSA_P256M_DRIVER_ENABLED` for more information.)
```

Comment:
This says "You need extra steps if you want to disable the built-in implementation of ECC algorithms" and "Refer to the documentation of `MBEDTLS_PSA_P256M_DRIVER_ENABLED` for more information" but I don't see those extra steps in `mbedtls_config.h`

Reply:
(preexisting, but ok, as long as this doesn't take too much back-and-forth) The steps are there, but it's true that they don't explicitly state that they let you “disable the built-in implementation”, they just explain what you can do without enabling the built-in implementation. I guess more verbiage won't hurt so I added another explanation that takes things the other way round.

Reply:
Note: `docs/driver-only-builds.md` is supposed to have all the details on how to build with a driver and without the built-in implementation in general. Ideally, we'd reference from the documentation of `MBEDTLS_PSA_P256M_DRIVER_ENABLED`, which would only need to explain the bits that are specific to this particular driver.

*If you find something is missing, please shout and I'll fix it as part of #7757.

Reply:
@mpg Are you reviewing this PR, or do we still need a second reviewer?

Reply:
I'm not reviewing, just had a quick look out of curiosity.

Reply:
(More precisely: don't count on me until I'm done with 8075 and 7757 - and by the time I'm finished with those, I hope this will have a 2nd reviewer already.)

Reply:
I will take second review now

Reply:
I'm going ahead and merging. Further documentation improvements are welcome but they aren't release-critical.